data/json/items/gun/shot.json

```diff
@@ -211,7 +236,11 @@
     "durability": 7,
     "barrel_length": 3,
     "magazine_well": 1,
-    "magazines": [ [ "shot", [ "saiga10mag", "saiga30mag" ] ] ],
+    "magazines": [
+      [ "shot", [ "saiga10mag", "saiga30mag" ] ],
+      [ "308", [ "falmag", "falbigmag" ] ],
```

Comment:
My error, they do exist - but they shouldn't. Again realism (if that's your thing due to no rifling) but also gameplay (everything doesn't need converting to everything else)